src/main/java/cd/go/contrib/elasticagent/KubernetesInstanceFactory.java

```diff
@@ -57,6 +57,7 @@ private KubernetesInstance create(CreateAgentRequest request, PluginSettings set
         container.setName(containerName);
         container.setImage(image(request.properties()));
         container.setImagePullPolicy("IfNotPresent");
+        container.setSecurityContext(new SecurityContextBuilder().withPrivileged(true).build());
```

Comment:
Shouldn't we provide an option if users don't want to start the container in a privileged mode?